comptests/prov/test_prov_k8s_cortx.py

```diff
@@ -431,3 +451,93 @@ def test_32654(self):
                                            data_disk_per_cvg=config['data_disk_per_cvg'],
                                            master_node_list=self.master_node_list,
                                            worker_node_list=self.master_node_list)
+
+    @pytest.mark.lc
+    @pytest.mark.comp_prov
+    @pytest.mark.tags("TEST-41569")
+    def test_41569(self):
+        """
+        S3-IO Operations.
+        """
+        LOGGER.info("Test Started.")
+        LOGGER.info("Step 1: Get Access and Secret Key.")
+        resp = self.master_node_obj.execute_cmd(cmd=commands.CMD_GET_ACCESS_KEY, read_lines=True)
+        access_key = "".join(resp).replace("\\n", "\n")
+        LOGGER.info(access_key)
+        resp = self.master_node_obj.execute_cmd(cmd=commands.CMD_GET_SECRET_KEY, read_lines=True)
+        secret_key = "".join(resp).replace("\\n", "\n")
+        LOGGER.info(secret_key)
+        LOGGER.info("Step 2: Fetch Interface IP, HTTP and HTTPS Ports.")
+        resp = system_utils.execute_cmd(cmd=commands.CMD_GET_IP_IFACE.format
+                                        (self.deploy_cfg['iface']))
+        LOGGER.info(resp)
+        eth1_ip = resp[1].strip("'\\n'b'")
+        LOGGER.info(eth1_ip)
+        if self.service_type == "NodePort":
+            resp = ext_lbconfig_utils.configure_nodeport_lb(self.master_node_list[0],
+                                                            self.deploy_cfg['iface'])
+            LOGGER.info(resp)
+            if not resp[0]:
```

Comment:
Can the testcase go ahead if resp[0] is none? If not please try to exit the test.

what is the expected value of resp[0], if it is not boolean or 0 then you might face the following exception
resp[0]
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
IndexError: list index out of range

pleae handle it if it is the case.